documentation/how-to-use-malware-detection-backend.md

```diff
@@ -0,0 +1,102 @@
+# Using malware detection backend
+
+If you are developing a service where users can upload files, you will probably want to check if these files are safe or not.
+
+The malware detection app is here to give you several backend implementations and others can be added to match your requirements.
+
+## Global idea
+
+A backend should inherit from `lasuite.malware_detection.backends.base.BaseBackend` class and implement the abstract methods declared in it.
+The backend service then is responsible to analyse the file and give an answer by calling a defined callback.
```

Comment:
Then the `BaseBackend` should define the `__init__` method with a `callback_path` kwarg ?